server/src/graphql/resolvers/column-resolvers.js

```diff
@@ -14,6 +14,9 @@ const schema = {
         deleteColumnById(root, { id }) {
             return dataSources.boardService.deleteColumnById(id)
         },
+        changeTaskOrderForOneColumn(root, args) {
```

Comment:
Shorter name could be 'changeTaskOrderInColumn'

Reply:
good! changed this.